.idea/encodings.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you should add the whole .idea folder to .gitignore so that it's not commited, as it's specific to your IDE